population_queries.sqlite

```diff
@@ -0,0 +1,37 @@
+-- This is the first query:
+SELECT DISTINCT year from population_years;
+
+-- Add your additional queries below:
+--The largest population size for Gabon in this dataset
+SELECT DISTINCT population
+FROM population_years
+WHERE country = "Gabon"
+ORDER BY population DESC
+LIMIT 1;
+
+--10 lowest population countries in 2005
+SELECT *
+FROM population_years
+WHERE year = 2005
+ORDER BY population ASC
+LIMIT 10;
+
+--All the distinct countries with a population of over 100 million in the year 2010
+SELECT DISTINCT country, population
+FROM population_years
+WHERE population > 100.0
+AND year = 2010
+ORDER BY population DESC;
+
+--How many countries in this dataset have the word "Islands" in their name
+SELECT COUNT(*) FROM (SELECT DISTINCT country
+FROM population_years
+WHERE country like '%Islands%');
```

Comment:
There should be a way to combine these multiple select statements into one. 
Maybe something like this.
`selectec DISTINCT country, count(*) from population_years where country like '%Island%');`

Reply:
@scriptmunkeeofficial thanks. I can update it to
SELECT COUNT(DISTINCT country)
FROM population_years
WHERE country like '%Islands%';